x-pack/plugins/alerting/common/routes/rule/response/schemas/v1.ts

```diff
@@ -204,37 +462,166 @@ export const alertDelaySchema = schema.object(
+    },
+  }),
+  name: schema.string({
+    meta: {
+      description: ' The name of the rule.',
+    },
+  }),
+  tags: schema.arrayOf(
+    schema.string({
+      meta: { description: 'The tags for the rule.' },
+    })
+  ),
+  rule_type_id: schema.string({
+    meta: { description: 'The rule type identifier.' },
+  }),
+  consumer: schema.string({
+    meta: {
+      description:
+        'The name of the application or feature that owns the rule. For example: `alerts`, `apm`, `discover`, `infrastructure`, `logs`, `metrics`, `ml`, `monitoring`, `securitySolution`, `siem`, `stackAlerts`, or `uptime`.',
+    },
+  }),
   schedule: intervalScheduleSchema,
   actions: schema.arrayOf(actionSchema),
   params: ruleParamsSchema,
   mapped_params: schema.maybe(mappedParamsSchema),
-  scheduled_task_id: schema.maybe(schema.string()),
-  created_by: schema.nullable(schema.string()),
-  updated_by: schema.nullable(schema.string()),
-  created_at: schema.string(),
-  updated_at: schema.string(),
-  api_key_owner: schema.nullable(schema.string()),
-  api_key_created_by_user: schema.maybe(schema.nullable(schema.boolean())),
-  throttle: schema.maybe(schema.nullable(schema.string())),
-  mute_all: schema.boolean(),
+  scheduled_task_id: schema.maybe(
+    schema.string({
+      meta: {
+        description: 'Identifier of the scheduled task.',
+      },
+    })
+  ),
+  created_by: schema.nullable(
+    schema.string({
+      meta: {
+        description: 'The identifier for the user that created the rule.',
+      },
+    })
+  ),
+  updated_by: schema.nullable(
+    schema.string({
+      meta: {
+        description: 'The identifier for the user that updated this rule most recently.',
+      },
+    })
+  ),
+  created_at: schema.string({
+    meta: {
+      description: 'The date and time that the rule was created.',
+    },
+  }),
+  updated_at: schema.string({
+    meta: {
+      description: 'The date and time that the rule was updated most recently.',
+    },
+  }),
+  api_key_owner: schema.nullable(
+    schema.string({
+      meta: {
+        description:
+          'The owner of the API key that is associated with the rule and used to run background tasks.',
+      },
+    })
+  ),
+  api_key_created_by_user: schema.maybe(
+    schema.nullable(
+      schema.boolean({
+        meta: {
+          description:
+            'Indicates whether the API key that is associated with the rule was created by the user.',
+        },
+      })
+    )
+  ),
+  throttle: schema.maybe(
+    schema.nullable(
+      schema.string({
+        meta: {
+          description:
+            'Deprecated in 8.13.0. Use the `throttle` property in the action `frequency` object instead. The throttle interval, which defines how often an alert generates repeated actions. NOTE: You cannot specify the throttle interval at both the rule and action level. If you set it at the rule level then update the rule in Kibana, it is automatically changed to use action-specific values.',
```

Comment:
>            'Deprecated in 8.13.0. Use the `throttle` property in the action `frequency` object instead...

I think you can also add `deprecated: true` per https://github.com/elastic/kibana/pull/181240